src/client.ts

```diff
@@ -235,10 +239,20 @@ export default class Client implements ClientInterface {
       vscode.commands.registerCommand(
         Command.Update,
         this.updateServer.bind(this)
-      )
+      ),
+      vscode.commands.registerCommand(Command.RunTest, this.runTest.bind(this))
     );
   }
 
+  private runTest(_path: string, _name: string, command: string) {
+    if (this.terminal === undefined) {
+      this.terminal = vscode.window.createTerminal({ name: "Run test" });
```

Comment:
Is the name mandatory? I'm wondering it makes sense to call it that, since someone could choose to keep the terminal open afterwards for general usage.